### PR TITLE
Update labs

### PR DESCRIPTION
**Pull request by ineiti**
The 2024 update is in!

Not sure how to review this. Check some of the descriptions. The links should be OK, as I used a link checker.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -92,21 +92,39 @@ url_path:
 
 ## Run the application locally
 
-Ensure you have the required tools to create Python virtual environments. You might need to install a specific package depending on your distribution, e.g. on Debian `python3-venv`:
+Ensure you have the required tools to create Python virtual environments.
+You can either install the dependencies globally, or use [devbox](https://www.jetpack.io/devbox).
+
+### Install dependencies globally
```
Comment (PascalinDe) [inline README.md:98]: "globally" usually means "for all users on the system", so I'd change that to "Install dependencies in a virtualenv"

File: README.md [hunk 2]
```diff
@@ -92,21 +92,39 @@ url_path:
 
 ## Run the application locally
 
-Ensure you have the required tools to create Python virtual environments. You might need to install a specific package depending on your distribution, e.g. on Debian `python3-venv`:
+Ensure you have the required tools to create Python virtual environments.
+You can either install the dependencies globally, or use [devbox](https://www.jetpack.io/devbox).
```
Comment (PascalinDe) [inline README.md:96]: same comment for "globally" as below

File: UPDATE.md [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+  - write an email to the professor
+    - remind them the goals of the Factory and the Showcase
+    - add a link to the projects of their lab
+    - ask them if it's OK like that or if they want to add/remove/modify any entry
+    - remind them that we do
+      - maturity evaluation
+      - promote projects
+      - work on specific projects
+      - -> perhaps we should have an article with some of the most typical work we did
+        for the labs
```
Comment (PascalinDe) [inline UPDATE.md:42]: is that a comment?

File: data/HexHive/projects.yaml [hunk 4]
```diff
@@ -483,15 +476,11 @@ projects:
...
       type: retired
       work: 2021/Q1 worked on improving and integrating with microG and published app
-      products:
-        - type: App
-          title: LibreCHovid
-          url: https://f-droid.org/en/packages/ch.corona.tracing/
     c4dt_contact:
       name: Christian Grigis
       email: christian.grigis@epfl.ch
     date_added: 2020-10-05
```
Comment (PascalinDe) [inline data/HexHive/projects.yaml:482]: shouldn't he be removed as contact since he's no longer part of C4DT?

File: data/labs.yaml [hunk 5]
```diff
@@ -56,10 +59,8 @@ labs:
       software updates, provide public randomness, enable privacy-conscious medical-data sharing and a lot more. Other projects include
       communicating securely over insecure channels and fast, scalable, accountable anonymous communication.
     contacts:
-      - name: Jeff Allen
-        email: jeff.allen@epfl.ch
-      - name: Philipp Svetolik Jovanovic
-        email: philipp.jovanovic@epfl.ch
+      - name: Pierluca Borso
```
Comment (PascalinDe) [inline data/labs.yaml:62]:
```suggestion
      - name: Pierluca Borsò-Tan
```

File: UPDATE.md (hunk 3 above)
Comment (ineiti) [reply UPDATE.md:42]: https://github.com/c4dt/showcase/issues/328

File: data/HexHive/projects.yaml (hunk 4 above)
Comment (ineiti) [reply data/HexHive/projects.yaml:482]: Removed this and other old contacts.

**Pull request merged**